### PR TITLE
remove legacy reference to 'extreme' option in docker cleanup process

### PR DESCRIPTION
**Pull request by claraberendsen**
### Description
This [issue](https://github.com/gazebo-tooling/release-tools/issues/929) appeared today due to legacy code expecting extreme option for the docker_cleanup process.

### CODE REVIEW

**Comment by claraberendsen** [comment]
This doesn't solve the issue is that disk usage is over 90 % on drogon. It seems like there is limitation to the memory docker_cleanup script can do.  We are seeing this issue in drogon because it's not a machine that gets reprovisioned constantly. However this would unblock us from keeping drogon off the buildfarm.

File: jenkins-scripts/docker/lib/boilerplate_prepare.sh [hunk 1]
```diff
@@ -43,13 +43,6 @@ if [[ -z ${DO_NOT_CHECK_DOCKER_DISK_USAGE} ]]; then
        source ${SCRIPT_DIR}/lib/docker_cleanup.sh high
     fi
```
Comment (j-rivero) [inline jenkins-scripts/docker/lib/boilerplate_prepare.sh:45]:
```suggestion
```

File: jenkins-scripts/docker/lib/docker_cleanup.sh [hunk 2]
```diff
@@ -28,7 +28,7 @@ docker_version="$(sudo docker version --format '{{.Server.APIVersion}}')"
             sudo docker system prune --all -f
             ;;
         *)
-            echo "Did not run docker cleaner, level $cleanup_level not recognized (accepted values low | medium | high | extreme!! "
+            echo "Did not run docker cleaner, level $cleanup_level not recognized (accepted values low | medium | high !! "
```
Comment (j-rivero) [inline jenkins-scripts/docker/lib/docker_cleanup.sh:31]:
```suggestion
            echo "Did not run docker cleaner, level $cleanup_level not recognized (accepted values low | medium | high )"
```

**Review by j-rivero — APPROVED** [review]
Couple of cosmetic issues. Looks good.

**Pull request merged**